src/types/result.d.ts

```diff
@@ -0,0 +1,21 @@
+export type Result = {
+  id: number,
+  stderr: string
+}
+
+export type TestcaseResult = { 
```

Comment:
use interfaces and OOP-style extension here too

Reply:
done